app/services/proofing/lexis_nexis/ddp/response_redacter.rb

```diff
@@ -0,0 +1,139 @@
+          emailage.emailriskscore.domainrelevantinfoid
+          emailage.emailriskscore.domainriskcountry
+          emailage.emailriskscore.domainrisklevel
+          emailage.emailriskscore.domainrisklevelid
+          emailage.emailriskscore.eaadvice
+          emailage.emailriskscore.eaadviceid
+          emailage.emailriskscore.eareason
+          emailage.emailriskscore.eareasonid
+          emailage.emailriskscore.eariskband
+          emailage.emailriskscore.eariskbandid
+          emailage.emailriskscore.eascore
+          emailage.emailriskscore.eastatusid
+          emailage.emailriskscore.emailexists
+          emailage.emailriskscore.first_seen_days
+          emailage.emailriskscore.firstverificationdate
+          emailage.emailriskscore.fraudrisk
+          emailage.emailriskscore.namematch
+          emailage.emailriskscore.phone_status
+          emailage.emailriskscore.responsestatus.errorcode
+          emailage.emailriskscore.responsestatus.status
+          emailage.emailriskscore.shipforward
+          emailage.emailriskscore.status
+          emailage.emailriskscore.totalhits
+          emailage.emailriskscore.uniquehits
+          enabled_services
+          event_datetime
+          event_type
+          fraudpoint.conversation_id
+          fraudpoint.friendly_fraud_index
+          fraudpoint.manipulated_identity_index
+          fraudpoint.product_status
+          fraudpoint.transaction_reason_code
+          fraudpoint.risk_indicators_codes
+          fraudpoint.risk_indicators_descriptions
+          fraudpoint.score
+          fraudpoint.stolen_identity_index
+          fraudpoint.suspicious_activity_index
+          fraudpoint.synthetic_identity_index
+          fraudpoint.transaction_status
+          fraudpoint.vulnerable_victim_index
+          integration_hub_results
+          org_id
+          policy
+          policy_details_api
+          policy_engine_version
+          policy_score
+          primary_industry
+          reason_code
+          request_duration
+          request_id
+          request_result
+          review_status
+          risk_rating
+          secondary_industry
+          service_type
+          session_id
+          session_id_query_count
+          summary_risk_score
+          tmx_reason_code
+          tmx_risk_rating
+          tmx_summary_reason_code
+          tmx_variables
+          tps_datetime
+          tps_duration
+          tps_error
+          tps_result
+          tps_type
+          tps_vendor
+          tps_was_timeout
+          unknown_session
+        ]
+
+        # @param [Hash] body
+        def self.redact(hash)
+          whitelisted_response_h = hash.slice(*ALLOWED_RESPONSE_FIELDS)
+          unwhitelisted_fields = hash.keys - whitelisted_response_h.keys
+          unwhitelisted_fields.each do |key|
+            whitelisted_response_h[key] = '[redacted]'
+          end
+          whitelisted_response_h
```

Comment:
Sorry if [earlier comments](https://github.com/18F/identity-idp/pull/6857#discussion_r957431021) and [suggestions](https://github.com/18F/identity-idp/pull/6857#discussion_r957822604) about names weren't clear. Let's rename these variables to not use `whitelist`

For context, we've been making a concerted effort in this codebase to move away from "whitelist/blacklist" because of some of the negative connotations of those words. See previous PRs and issues (#5361, https://github.com/18F/identity-devops/pull/2538)

```suggestion
          filtered_response = hash.slice(*ALLOWED_RESPONSE_FIELDS)
          unfiltered_response = hash.keys - filtered_response.keys
          unfiltered_response.each do |key|
            filtered_response[key] = '[redacted]'
          end
          filtered_response
```